cmd/fyne/internal/mobile/bind.go

```diff
@@ -69,6 +69,11 @@ func packagesConfig(targetOS string) *packages.Config {
 	config := &packages.Config{}
 	// Add CGO_ENABLED=1 explicitly since Cgo is disabled when GOOS is different from host OS.
 	config.Env = append(os.Environ(), "GOARCH=arm", "GOOS="+targetOS, "CGO_ENABLED=1")
+	if targetOS == "android" {
+		// with Cgo enabled we need to ensure the C compiler is set via CC to
+		// avoid the error: "gcc: error: unrecognized command line option '-marm'"
+		config.Env = append(os.Environ(), androidEnv["arm"]...)
```

Comment:
Do we need the env additions from line 71?
```suggestion
		config.Env = append(config.Env, androidEnv["arm"]...)
```

Reply:
No, it is not needed. `anroidEnv` contains already all env settings from line 71: https://github.com/fyne-io/fyne/blob/master/cmd/fyne/internal/mobile/env.go#L116-L122
Adding will duplicate them.